js/apps/admin-ui/src/identity-providers/add/OIDCGeneralSettings.tsx

```diff
@@ -16,12 +16,15 @@ export const OIDCGeneralSettings = ({ id }: { id: string }) => {
 
   const {
     register,
+    control,
     formState: { errors },
   } = useFormContext();
 
+  const alias = useWatch({control, name: "alias"});
+
   return (
     <>
-      <RedirectUrl id={id} />
```

Comment:
Since `id` is no longer used it can be removed from the props for this component.